docs/src/pages/components/links/UnderlineLink.tsx

```diff
@@ -1,27 +1,21 @@
-/* eslint-disable jsx-a11y/click-events-have-key-events */
-/* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import * as React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Link from '@material-ui/core/Link';
+import * as CSS from 'csstype';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      ...theme.typography.body1,
-      '& > * + *': {
-        marginLeft: theme.spacing(2),
-      },
-    },
-  }),
-);
+const Box = styled('div')(({ theme }) => ({
+  ...(theme.typography.body1 as CSS.Properties),
+  '& > :not(style) + :not(style)': {
+    marginLeft: theme.spacing(2),
+  },
+}));
 
 export default function UnderlineLink() {
-  const classes = useStyles();
   const preventDefault = (event: React.SyntheticEvent) => event.preventDefault();
 
   return (
-    <div className={classes.root} onClick={preventDefault}>
+    <Box onClick={preventDefault}>
```

Comment:
It overflows on mobile. https://deploy-preview-25303--material-ui.netlify.app/components/links/#underline. I think that we need a flex wrap

Reply:
sure! I didn't realize this until now. I review tomorrow.

Reply:
Resolved